src/request/controllers/create_request_controller.py

```diff
@@ -0,0 +1,49 @@
+from api.utils import failure_response
+from api.utils import success_response
+from person.models import Person
+from rest_framework import status
+from ride.models import Ride
+
+from ..models import Request
+
+
+class CreateRequestController:
+    def __init__(self, request, data, serializer):
+        self._request = request
+        self._data = data
+        self._serializer = serializer
+
+    def process(self):
+        approvee_id = self._data.get("approvee_id")
+        ride_id = self._data.get("ride_id")
+
+        # Verify all required information is provided
+        if approvee_id is None or ride_id is None:
+            return failure_response("Missing request information", 400)
+
+        approvee_exists = Person.objects.filter(id=int(approvee_id)).exists()
```

Comment:
why is the approvee_id casted to an int? shouldn't you just specify in the API that the POST body's approvee_id field must be an int? In which case, for safety, you would probably want to also check if the required information is the correct type.